src/app/public/modules/shared/initialization.service.ts

```diff
@@ -6,13 +6,38 @@ import {
   BBHelpClient
 } from '@blackbaud/help-client';
 
+import {
+  SkyAppWindowRef
+} from '@skyux/core';
+
+import {
+  SkyAppConfig
+} from '@skyux/config';
+
 import {
   HelpWidgetConfig
 } from './widget-config';
 
 @Injectable()
 export class HelpInitializationService {
-  public load(config: HelpWidgetConfig) {
-      return BBHelpClient.load(config);
+  public constructor(private windowRef: SkyAppWindowRef, private config: SkyAppConfig) {
```

Comment:
i deemed this a minor change and not a breaking change. it would break if you were creating this svc via `new HelpInitializationService`, but no one should be doing that. if you're using it as intended, i.e. as an angular provider in a sky spa, you will have these two dependencies right? and thus this wouldn't be breaking. thoughts?

as for the behavior change in the `load`, that won't break anything. it could overwrite certain config values, but it's overwriting to what the "correct" values of them should be. so that feels like an enhancement, hence why i went w/ a _minor_ version bump.

im not married to that logic tho.

Reply:
Should be fine introducing that change, but would you mind formatting following https://docs.blackbaud.com/skyux-team-docs/style#constructors?